Week-6/W6Exercise6.3/Index.js

```diff
@@ -0,0 +1,25 @@
+const sortArr = (arr) => {
+    let low = 0, high = arr.length - 1, mid = 0, temp = 0;
+    while (mid <= high) {
+        if (arr[mid] === 0) {
+            temp = arr[mid];
```

Comment:
can you think of swapping two elements in js in a better way?

Reply:
`[arr[mid], arr[low]] = [arr[low], arr[mid]]`

Would use this from now on